.circleci/config.yml

```diff
@@ -100,7 +100,7 @@ jobs:
 
   unit_tests:
     <<: *job_template
-    resource_class: xlarge
+    resource_class: large
```

Comment:
I'm curious: while we don't _need_ it anymore thanks to the memory gains, did we check if using `xlarge` resources results in a shorter job duration? If yes, it might be worth keeping.